pkg/apis/keycloak/v1alpha1/keycloakrealm_types.go

```diff
@@ -7,6 +7,10 @@ import (
 // KeycloakRealmSpec defines the desired state of KeycloakRealm.
 // +k8s:openapi-gen=true
 type KeycloakRealmSpec struct {
+	// When set to true, this KeycloakRealm will be marked external and not be managed by this operator.
+	// It can then be used to create clients and users.
+	// +optional
+	External bool `json:"external,omitempty"`
```

Comment:
This is an interesting flag.... Could be made more generic. Would there ever be a case where you want to have keycloak manage a realm for a while, then stop but still be able to manage clients? If so, then maybe the flag should be:
Unmanaged bool
Managed external may make sense as well.